joomla/joomla-cms
# Update default.php

Pull Request for Issue # .

### Summary of Changes

Check for the emptiness of $urls on lines 98 and 132 has been removed since the variable $urls can never be empty.

### Testing Instructions

Create an article and check whether it is properly displayed or not with its url.

### Actual result BEFORE applying this Pull Request

![Untitled](https://user-images.githubusercontent.com/76894046/110879081-5e49cc00-8302-11eb-97c9-21617796edc3.jpg)


### Expected result AFTER applying this Pull Request

![Untitled](https://user-images.githubusercontent.com/76894046/110879096-6570da00-8302-11eb-92cc-e3722e491a1d.jpg)


### Documentation Changes Required

None
